solr/core/src/java/org/apache/solr/security/PKIAuthenticationPlugin.java

```diff
@@ -374,17 +374,13 @@ PublicKey fetchPublicKeyFromRemote(String nodename) {
     }
   }
 
-  @Override
-  public void setup(Http2SolrClient.Builder httpClientBuilder, Http2SolrClient client) {
-    setup(client, httpClientBuilder);
-  }
-
   @Override
   public void setup(Http2SolrClient client) {
-    setup(client, null);
+    setup(null, client);
   }
 
-  private void setup(Http2SolrClient client, Http2SolrClient.Builder builder) {
+  @Override
+  public void setup(Http2SolrClient.Builder httpClientBuilder, Http2SolrClient client) {
```

Comment:
Suggest naming this param "builder" as it was, or "httpSolrClientBuilder".  Note "client" is simply "client".